docs/fast_data/single_view_trigger_generator.md

```diff
@@ -8,29 +8,59 @@ sidebar_label: Single View Trigger Generator
 This Plugin is a BETA Plugin and, as such, is currently under active development. Pay attention using it.
 :::
 
-The Single View Trigger Generator is a new service of the Fast Data architecture.
-Its main job is to execute the strategies which are currently handled by the [Real Time Updater](/fast_data/realtime_updater.md).
+The Single View Trigger Generator is a new component of the Fast Data architecture. Its introductions allow to split the import phase
```

Comment:
I would add that this component enables the full event-driven architecture and not only to split ownership of strategies between components

Reply:
Done